src/libsystemd-network/sd-dhcp-client.c

```diff
@@ -150,6 +153,33 @@ static int client_receive_message_udp(
                 void *userdata);
 static void client_stop(sd_dhcp_client *client, int error);
 
+int sd_dhcp_client_id_to_string(char **ret, const void *data, size_t len) {
+        const sd_dhcp_client_id *client_id = data;
+
+        switch (client_id->type) {
+        case 0:
```

Comment:
Don't we have constants defined for those numbers?

Reply:
`sd_dhcp_client_set_client_id()`  is called once with ARPHRD_ETHER, but I don't think that enum fits well.